MathlibTest/unify_denoms.lean

```diff
@@ -0,0 +1,38 @@
+import Mathlib
+
+open BigOperators
+open Finset
+
+example (n : ℕ ) : n * (n + 1) / 2 + n + 1 = (n + 1) * (n + 2) / 2 := by
+  unify_denoms
+  ring_nf
+
+example (n : ℕ) : ∑ i ∈ Icc 0 n, i = (n * (n + 1)) / 2 := by
+  induction n with
+  | zero => simp
+  | succ n hn =>
+    rw [Finset.sum_Icc_succ_top, hn]
+    unify_denoms
+    ring_nf
+
+example (n : ℕ) : ∑ i ∈ Icc 0 n, i ^ 2 = (n * (n + 1) * (2* n + 1)) / 6 := by
+  induction n with
+  | zero => simp
+  |succ n hn =>
+    rw [Finset.sum_Icc_succ_top, hn]
+    unify_denoms
+    ring_nf
```

Comment:
Can you replace these tests by the precise computation that is done by `unify_denoms`?